openpype/hosts/blender/plugins/publish/extract_thumbnail.py

```diff
@@ -116,10 +116,19 @@ def process(self, instance):
         instance.data.setdefault("representations", [])
 
         tags = ["thumbnail", "review"]
+
+        # Removed `review` tags from thumbnail if current subset instance
+        # already has a representation with `review` tag or if there is a
+        # `Review` subset in the current publish context.
         for repre in instance.data["representations"]:
             if "review" in repre.get("tags", []):
                 tags.remove("review")
                 break
+        else:
+            for context_instance in instance.context:
+                if context_instance.data.get("family") == "review":
```

Comment:
simply `context` is clearer.
```suggestion
            for context in instance.context:
                if context.data.get("family") == "review":
```

Reply:
This is the way to check all instances of the context.

The context is not a list of contexts but a list of instances. 
[https://api.pyblish.com/pyblish.api/context](https://api.pyblish.com/pyblish.api/context)

And there is only one context entity, `instance.context` refere to the parent context.
[https://api.pyblish.com/pyblish.api/instance/instance.context](https://api.pyblish.com/pyblish.api/instance/instance.context)

Reply:
Could be replaced with `sibling_instance` but `instance in instance.context` is True so `context_instance ` seems not so confusing.

Reply:
Okay I got it wrong in the first place. But if you have to do that, isn't it smarter to turn it into a `ContextPlugin` and do for the whole block: 
```suggestion
        for instance in context:
            if instance.data.get("family") == "review":
                tags.remove("review")
                break
            else:
                for repre in instance.data["representations"]:
                    if "review" in repre.get("tags", []):
                        tags.remove("review")
                        break
```
This could also be faster, because it'll be run only once, when with your code it is run for every collected instance.

Reply:
Okey, this will move to the next PR, as we don't have tag filters in integrated_kitsu_review the most importante changement is to not generate thumbnail for review. So this feature is reverted from this PR, removing `review` from the families is enough.